pkg/reconciler/revision/resources/queue.go

```diff
@@ -68,6 +70,11 @@ var (
 		ContainerPort: int32(networking.UserQueueMetricsPort),
 	}}
 
+	profilingPort = corev1.ContainerPort{
```

Comment:
Yeah. That would be possible but since all the other port variables are defined in this place (even if they're used just once) I'd prefer having them side by side here where they are now. wdyt?